### ISSUE TITLE
Ionic 3 Error: Metadata mismatch for module /ion-affix

### ISSUE THREAD

**Issue by UltimateGemini** [jonaszuberbuehler/ion-affix#23]
@jonaszuberbuehler  When i try to run `ionic cordova run android --prod` I get this error of:
Error: Metadata version mismatch for module /node_modules/ion-affix/dist/ion-affix.module.d.ts, found version 4, expected 3

I tried up do npm update in the package.json but it didnt do anything. I also removed node_modules and ran npm install and still the same thing. Any suggestions?

**Comment by jonaszuberbuehler** [comment]
@Noel-F Can you post your package.json plz?

**Comment by UltimateGemini** [comment]
@jonaszuberbuehler 
```
{
  "name": "MYAPP",
  "version": "0.0.1",
  "author": "Ionic Framework",
  "homepage": "http://ionicframework.com/",
  "private": true,
  "scripts": {
    "clean": "ionic-app-scripts clean",
    "build": "ionic-app-scripts build",
    "lint": "ionic-app-scripts lint",
    "ionic:build": "ionic-app-scripts build",
    "ionic:serve": "ionic-app-scripts serve"
  },
  "dependencies": {
    "@angular/common": "4.4.3",
    "@angular/compiler": "4.4.3",
    "@angular/compiler-cli": "4.4.3",
    "@angular/core": "4.4.3",
    "@angular/forms": "4.4.3",
    "@angular/http": "4.4.3",
    "@angular/platform-browser": "4.4.3",
    "@angular/platform-browser-dynamic": "4.4.3",
    "@ionic-native/core": "4.3.2",
    "@ionic-native/splash-screen": "4.3.2",
    "@ionic-native/status-bar": "4.3.2",
    "@ionic/storage": "2.1.3",
    "angularfire2": "^4.0.0-rc0",
    "cordova-android": "6.3.0",
    "cordova-ios": "4.5.4",
    "cordova-plugin-device": "^1.1.7",
    "cordova-plugin-splashscreen": "^4.1.0",
    "cordova-plugin-whitelist": "^1.3.3",
    "firebase": "^3.9.0",
    "ion-affix": "^1.1.1",
    "ionic-angular": "3.9.2",
    "ionic-plugin-keyboard": "^2.2.1",
    "ionicons": "3.0.0",
    "rxjs": "5.5.2",
    "sw-toolbox": "3.6.0",
    "zone.js": "0.8.18"
  },
  "devDependencies": {
    "@ionic/app-scripts": "3.1.8",
    "typescript": "2.4.2"
  },
  "description": "An Ionic project",
  "cordova": {
    "plugins": {
      "ionic-plugin-keyboard": {},
      "cordova-plugin-whitelist": {},
      "cordova-plugin-device": {},
      "cordova-plugin-splashscreen": {}
    },
    "platforms": [
      "android",
      "ios"
    ]
  }
}
```

**Comment by jonaszuberbuehler** [comment]
@Noel-F Sorry for my late reply. Your issue should be solved by updating the angular deps to 5.0.x.

**Comment by jonaszuberbuehler** [comment]
Closing due to inactivity.
